sonar-scm-svn-plugin/src/main/java/org/sonar/plugins/scm/svn/SvnScmProvider.java

```diff
@@ -54,4 +70,39 @@ public boolean supports(File baseDir) {
   public BlameCommand blameCommand() {
     return blameCommand;
   }
+
+  @Nullable
+  @Override
+  public Set<Path> branchChangedFiles(String targetBranchName, Path rootBaseDir) {
+    SVNClientManager clientManager = null;
+    try {
+      clientManager = newSvnClientManager(configuration);
+      SVNWCClient wcClient = clientManager.getWCClient();
+      SVNInfo svnInfo = wcClient.doInfo(rootBaseDir.toFile(), null);
+      String base = "/" + Paths.get(svnInfo.getRepositoryRootURL().getPath()).relativize(Paths.get(svnInfo.getURL().getPath()));
+
+      SVNLogClient svnLogClient = clientManager.getLogClient();
+      Set<Path> paths = new HashSet<>();
+      svnLogClient.doLog(new File[] {rootBaseDir.toFile()}, null, null, null, true, true, 0, svnLogEntry -> {
+        svnLogEntry.getChangedPaths().values().forEach(entry -> {
+          if (entry.getCopyPath() == null) {
```

Comment:
Are you correctly filtering deleted files?

Reply:
Good catch, fixed (for Git too).